src/Analyser/MutatingScope.php

```diff
@@ -674,6 +676,10 @@ private function getNodeKey(Expr $node): string
 			$key .= '/*' . $node->getAttribute('startFilePos') . '*/';
 		}
 
+		if ($node->hasAttribute(self::KEEP_VOID_ATTRIBUTE_NAME)) {
```

Comment:
```suggestion
		if ($node->getAttribute(self::KEEP_VOID_ATTRIBUTE_NAME) === true) {
```
Let's see if I can do some phone fixing again.. :)